### DISCUSSION TITLE
Rate limiter

### DISCUSSION THREAD

**Discussion by janacm** [axios/axios#4608] [Q&A]
Anyone have a suggestion for limiting the rate of several outgoing Axios API calls that I make to an external API? I looked at https://github.com/aishek/axios-rate-limit/issues but there are many issues on it and doesn't have recent updates. 

I need to make about 100k API requests against an API that has a limit of 1000/minute.

**Comment by jasonsaayman** [comment]
You should implement a circuit breaker pattern, when limited by the API you either re-try when available or switch to another API

**Comment by aishek** [comment]
I decided to continue support of the https://github.com/aishek/axios-rate-limit, feel free to use it
